### DISCUSSION TITLE
How to toggle class in CDN Version?

### DISCUSSION THREAD

**Discussion by chanayus** [tw-in-js/twind#481] [Q&A] [answered]
Hello, My Project is vanilla static HTML, and I install Twind By CDN.
I want to toggle the "bg-red-500" class by clicking the button, but it's not working.

```html
<button id="btn" class="bg-red-500">Button</button>
````

```javascript
  const btn = document.querySelector("#btn")
  
  btn.addEventListener("click", () => {
      btn.classList.toggle("bg-red-500")
  })
````

and I have inspect elements with devtools, and I found the button classname is not the same in my code.

```html
<button id="btn" class="#17tpr9i">Button</button>
````

Anyone who know how to toggle class? 🙏

**Comment by rschristian** [comment, marked as answer]
Twind hashes classnames by default. You can easily configure this however:

```html
<script src="https://cdn.twind.style" crossorigin></script>
<script>
    twind.install({
        hash: false,
    });
</script>
```

This disables hashing, so `bg-red-500` will stay `bg-red-500`.

**Reply by chanayus** [reply]
Thank you so much. 🙏
